aea/aea_builder.py

```diff
@@ -970,6 +965,7 @@ def add_component_instance(self, component: Component) -> "AEABuilder":
         You will have to `reset()` the builder before calling `build()` again.
 
         :param component: Component instance already initialized.
+        :return: self
         """
         self._to_reset = True
         self._check_can_add(component.configuration)
```

Comment:
below, `set_context_namespace` does not state the return value

Reply:
Single line docstrings are permitted and not parsed. Without this we'd have to unnecessarily add an extensive docstring for each property.